### PR TITLE
CloudReady shim 15.4 submission

### PR DESCRIPTION
**Pull request by pnardini**
These are the artifacts that are required to submit
CloudReady's build of shim 15.4 for review.  This shim submission is
necessary to fully remediate the BootlHole vulnerability from 2020 and
revokes vulnerable versions of GRUB by using a new EV code signing cert
for the first time.

Shim binaries, build.log with redacted AWS credentials and the required,
fully filled-out README.md are all present.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -90,27 +114,33 @@ upstream commit 75b0cea7bf307f362057cc778efe89af4c615354 applied ?
...
+If the shim binaries can't be reproduced using the provided Dockerfile, please explain why that's the case and the differences would be.
 -------------------------------------------------------------------------------
-This repo contains the Dockerfile we use to build shim: https://github.com/neverware/shim-build/tree/v4
+All shim binaries can be built using our Dockerfile and instructions in the README.md of https://github.com/neverware/shim-build/tree/v5
 
 -------------------------------------------------------------------------------
 Which files in this repo are the logs for your build?   This should include logs for creating the buildroots, applying patches, doing the build, creating the archives, etc.
 -------------------------------------------------------------------------------
-[build.log](build.log)
+build.log
```
Comment (nicholasbishop) [inline README.md:141]: Lost the build link here

File: README.md [hunk 2]
```diff
@@ -90,27 +114,33 @@ upstream commit 75b0cea7bf307f362057cc778efe89af4c615354 applied ?
...
 
 -------------------------------------------------------------------------------
 Add any additional information you think we may need to validate this shim
 -------------------------------------------------------------------------------
-Our shim-15.1 build is largely upstream's 15.1 with our new public certificate embedded.  This build has been tested with our new grub, which is now up-to-date with fedora-33.
-
-
-
+We made our last shim submission as Neverware.
```
Comment (nicholasbishop) [inline README.md:146]: Might want to add links here to https://github.com/rhboot/shim-review/issues/27 and https://github.com/rhboot/shim-review/issues/106

File: README.md [hunk 3]
```diff
@@ -18,64 +20,86 @@ Here's the template:
...
 -------------------------------------------------------------------------------
 Who is the primary contact for security updates, etc.
 -------------------------------------------------------------------------------
 - Name: Nicholas Bishop
-- Position: software developer
-- Email address: nbishop@neverware.com
-- PGP key, signed by the other security contacts, and preferably also with signatures that are reasonably well known in the linux community: https://github.com/neverware/shim-review/blob/master/nbishop.key
+- Position: Software Engineer
+- Email address: nicholasbishop@google.com
+- PGP key, signed by the other security contacts, and preferably also with signatures that are reasonably well known in the Linux community: https://github.com/neverware/shim-review/blob/master/nbishop.key
```
Comment (nicholasbishop) [inline README.md:41]: Does the pgp key include the email address? We might need to update them if so

Comment (pnardini) [reply README.md:41]: Yep, seems like they do.  Will need a new key from you.

File: README.md (hunk 1 above)
Comment (pnardini) [reply README.md]: Added build link.

File: README.md (hunk 2 above)
Comment (pnardini) [reply README.md]: Added links to both of the above.

File: README.md (hunk 3 above)
Comment (pnardini) [reply README.md:41]: Will add new PGP keys in separate PR(s)

**Review by nicholasbishop — APPROVED** [review]

**Pull request merged**